### PR TITLE
Property group missing properties placeholders

### PR DESCRIPTION
**Pull request by kilbee**
Added placeholders for missing properties to be able to save and load new presets.

### CODE REVIEW

**Comment by ekaj2** [comment]
This is the wrong way to work around this IMHO. These props aren't used anymore AFAIK, so they should just be removed from the preset code.

**Comment by kilbee** [comment]
Ok, commented out conflicting properties as a quick fix - "AFAIK" sounds like an assumption, not a statement ;p so I'll leave cleaning up/adding props for someone who knows exactly which props are/will be used.
Awesome plugin btw.

**Comment by ekaj2** [comment]
@kilbee I'm just trying to be nice ;) There isn't really any reason to keep these properties around. When Maxime updated the unwrapping, he removed the first three, but I'm not sure that the last was intentional. Go ahead and remove them all completely, though, because we have all of our history on GitHub and there's no hurt in doing so. If you can do that, I'll merge the pull request and tag a new release. Thanks for pointing this one out and working on it!

@MaximeHerpin Should there be a twig leaf material prop?

**Comment by MaximeHerpin** [comment]
Deleting the twig leaf material property was not a mistake as the leaf object is now a choice of the user.

**Comment by ekaj2** [comment]
Good, just making sure :-)

**Comment by kilbee** [comment]
Ok then, props removed.

File: presets.py [hunk 1]
```diff
@@ -252,9 +244,9 @@ def execute(self, context):
             # verify that a colon is in the line to avoid an error with line.split(":")
             if ":" in line:
                 setting, value = line.split(":")
-                if setting == 'finish_unwrap':
-                    mtree_props.finish_unwrap = bool(int(value))
-                elif setting == "preserve_trunk":
+                #if setting == 'finish_unwrap':
+                #    mtree_props.finish_unwrap = bool(int(value))
+                if setting == "preserve_trunk":
```
Comment (ekaj2) [inline presets.py]: This needs to be completely removed...

File: presets.py [hunk 2]
```diff
@@ -298,14 +290,14 @@ def execute(self, context):
                     mtree_props.create_armature = bool(int(value))
                 elif setting == "bones_iterations":
                     mtree_props.bones_iterations = int(value)
-                elif setting == "visualize_leafs":
-                    mtree_props.visualize_leafs = bool(int(value))
+                #elif setting == "visualize_leafs":
+                #    mtree_props.visualize_leafs = bool(int(value))
```
Comment (ekaj2) [inline presets.py]: see above comment

File: presets.py [hunk 3]
```diff
@@ -298,14 +290,14 @@ def execute(self, context):
...
                 elif setting == "leafs_iteration_length":
                     mtree_props.leafs_iteration_length = int(value)
                 elif setting == "uv":
                     mtree_props.uv = bool(int(value))
-                elif setting == "unwrap_end_iteration":
-                    mtree_props.unwrap_end_iteration = int(value)
+                #elif setting == "unwrap_end_iteration":
+                #    mtree_props.unwrap_end_iteration = int(value)
```
Comment (ekaj2) [inline presets.py]: see above comment

File: presets.py [hunk 4]
```diff
@@ -330,8 +322,8 @@ def execute(self, context):
                     mtree_props.leaf_size = float(value)
                 elif setting == "leaf_chance":
                     mtree_props.leaf_chance = float(value)
-                elif setting == "twig_leaf_material":
-                    mtree_props.twig_leaf_material = value.replace("\n", "")
+                #elif setting == "twig_leaf_material":
+                #    mtree_props.twig_leaf_material = value.replace("\n", "")
```
Comment (ekaj2) [inline presets.py]: see above comment

**Pull request merged**